transports/webrtc/src/tokio/transport.rs

```diff
@@ -415,10 +412,7 @@ fn parse_webrtc_listen_addr(addr: &Multiaddr) -> Option<SocketAddr> {
     let port = iter.next()?;
     let webrtc = iter.next()?;
 
-    let port = match (port, webrtc) {
-        (Protocol::Udp(port), Protocol::WebRTCDirect) => port,
-        _ => return None,
-    };
+    let (Protocol::Udp(port), Protocol::WebRTCDirect) = (port, webrtc) else { return None };
```

Comment:
I am wondering if it wouldn't be cleaner to inline the pattern matching into the lines 412 and 413? We would have to say `else { return None; }` on each line but I think that reads actually quite well with the formatter putting it on one line.